.palantir/revapi.yml

```diff
@@ -1145,6 +1145,22 @@ acceptedBreaks:
       new: "method org.apache.iceberg.BaseMetastoreOperations.CommitStatus org.apache.iceberg.BaseMetastoreTableOperations::checkCommitStatus(java.lang.String,\
         \ org.apache.iceberg.TableMetadata)"
       justification: "Removing deprecated code"
+  "1.7.0":
+    org.apache.iceberg:iceberg-core:
+    - code: "java.method.returnTypeChanged"
```

Comment:
we can't change the return type of existing methods as it's breaking existing APIs. We need to follow the same approach I mentioned earlier (deprecated + introduce new methods with the new return type)

Reply:
done!